COVID19/src/utils/constans.js

```diff
@@ -19,13 +20,17 @@ const constans = {
   loginGreeting: 'Please, login and stay safe!',
   login: 'Login',
   forgotPW: 'Forgot password?',
+  errorLoginFail: 'Login failed!',
   urlAPI: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_in',
   urlAPIregIncident: 'https://hidden-cliffs-21927.herokuapp.com/api/incident',
+  urlSingOut: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_out',
```

Comment:
```suggestion
  urlAPI: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_in',
  urlAPIregIncident: 'https://hidden-cliffs-21927.herokuapp.com/api/incident',
  urlSingOut: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_out',
```

Only should be only one route and concat the path that you want access for example 

```
urlApi: 'https://hidden-cliffs-21927.herokuapp.com/api/'
```

if you want access to path only concat the route for example 

```
`${urlApi}user/sign_in`
```

Reply:
You are absolutely right, I like the idea ... now we solve it, I just have to do a replacement in the scripts that are involved.